worldengine/simulations/erosion.py

```diff
@@ -48,12 +48,12 @@ def is_applicable(self, world):
         return world.has_precipitations()
 
     def execute(self, world, seed):
-        water_flow = numpy.zeros((world.width, world.height))
-        water_path = numpy.zeros((world.width, world.height), dtype=int)
+        water_flow = numpy.zeros((world.height, world.width))
+        water_path = numpy.zeros((world.height, world.width), dtype=int)
         river_list = []
         lake_list = []
-        river_map = numpy.zeros((world.width, world.height))
-        lake_map = numpy.zeros((world.width, world.height))
+        river_map = numpy.zeros((world.height, world.width))
```

Comment:
This is where the great xy-swapping of 2015 started.^^
I think this is an important change since it is confusing at best when in ```drawing_functions.py``` there can be code like this:
```python
if world.ocean[y, x]:
    world.river_map[x, y] = 5  # this is no ingenious trick; just different coordinate systems
```